taxi/templates/base.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	{% load static %}
+		{% block title %}
+			<title>Taxi Service</title>
+		{% endblock %}
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
```

Comment:
The integrity attribute for the Bootstrap CSS link seems incorrect. Please verify and update the integrity hash to match the Bootstrap version you are using. This is important for ensuring the stylesheet loads securely.